src/Redist/libtorch-cuda-11.1/libtorch-cuda-11.1.proj

```diff
@@ -217,7 +217,7 @@
                     var outBytes = new byte[Size];
                     using (var accessor = mmf.CreateViewAccessor(Start, Size))
                     {
-                        accessor.ReadArray<byte>(0, outBytes, 0, Size);
+                        accessor.ReadArray<byte>(0, outBytes, 0, (int)Size);
```

Comment:
This cast is going to silently lose data: `i64` 0x1_0000_0001 (4GB+1byte) will be turned into just 0x1 `i32`

Reply:
It's ok as size is always < 250MB, it could have been kept as int32